src/GafferScene/RenderController.cpp

```diff
@@ -651,6 +651,11 @@ class RenderController::SceneGraph
 				m_objectInterface = renderer->object( name, object.get(), attributesInterface( renderer ) );
 			}
 
+			// Object computation may take a while so we're sensitive to
```

Comment:
Actually, this comment is misleading, given the subtlety of the situation. The old code was absolutely fine with respect to cancellation of object computations (line 566). The real issue we're fixing is that the _renderer_ methods might throw Cancelled. And actually, that's completely unexpected because the renderer methods don't take a Canceller argument. The real problem is the leaking through of a Canceller because we have a visualiser which is erroneously using the current context. Unless/until we adapt the Renderer interface to allow cancellation, I think there's a very good case for stopping the visualiser from ever throwing.